README.md

```diff
@@ -34,6 +34,7 @@ in a single K8s cluster.
 * rhoas CLI (https://github.com/redhat-developer/app-services-cli)
 * A user with administrative privileges in the OpenShift cluster and is logged in using `oc` or `kubectl`
 * brew coreutils (Mac only)
+* [yq][yq] if `kas-fleet-manager-service-template-params` is provided
```

Comment:
Yes, you're right. Theoretically, it is not a prerequisite. But since we've provided an example `kas-fleet-manager-service-template-params` script with `yq` command, I think most people will just adopt it directly. 

But I'm fine if you think I should remove this line from prerequisite list. Thanks.